src/AddTodoForm.js

```diff
@@ -1,10 +1,20 @@
+
 import React from 'react';
 
-export default function AddTodoForm() {
+export default function AddTodoForm({ onAddTodo }) {
+
+    function handleAddTodo(e) {
+        e.preventDefault()
+        const todoTitle = e.target.title.value
+        onAddTodo(todoTitle)
+        e.target.title.value = " "
+        console.log(todoTitle)
```

Comment:
Nice job here man!!! 💯 

Next time check out `form.reset()`. I think you'll figure out how to use it here :)